app/javascript/crayons/MarkdownToolbar/MarkdownToolbar.jsx

```diff
@@ -132,10 +175,45 @@ export const MarkdownToolbar = ({ textAreaId }) => {
       markdownSyntaxFormatters[syntaxName].getFormatting(textArea);
 
     textArea.value = newTextAreaValue;
+    textArea.dispatchEvent(new Event('input'));
     textArea.focus({ preventScroll: true });
     textArea.setSelectionRange(newCursorStart, newCursorEnd);
   };
 
+  const handleImageUploadStarted = () => {
+    const { textBeforeSelection, textAfterSelection, selectionEnd } =
+      getSelectionData(textArea);
+
+    const placeholder = '![Uploading image](...)';
```

Comment:
Refactor (blocking): Localize the placeholder text. It's used in at least two places in the code.

Reply:
If this is a blocker I can add it, but it would require a change to what translations we load into the frontend (currently we only include those in the `core` namespace, but this would live in `views.editor`:

https://github.com/forem/forem/blob/09828853f6cc0e07da74f4da885dfa93f5204662/app/views/layouts/application.html.erb#L111

I feel like doing that should be a separate PR - what do you think?

If we left that for later i18n work, I could just consolidate the couple of occurrences of this string into a constant for now